src/Promitor.Agents.ResourceDiscovery/Dockerfile.windows

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:7.0.402 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0.406 AS build
```

Comment:
Can you update the Linux image as well please?

Reply:
Forgot to commit it sorry :)

Reply:
No worries!